capstone/src/components/User/Adventurer/FindAdventure/List/List.jsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from 'react';
+import { VStack, Box, Badge, Heading } from '@chakra-ui/react';
+import AdventurerContext from '../../../../../Contexts/AdventurerContext';
+
+export default function List() {
+  const { isDataFetched, restaurants } = useContext(AdventurerContext);
+  if (isDataFetched) {
+    return (
+      <VStack>
+        {restaurants.map((restaurant, index) => (
+          <Box key={index} shadow="md" width="60%" m="10px !important" display="flex">
```

Comment:
A stack in Chakra looks like it is used to provide spacing to its children, which is why the margin is being overridden *(see the default .5rem [here](https://github.com/chakra-ui/chakra-ui/blob/main/packages/layout/src/stack.tsx#L106))*.  You might try to set the spacing on the VStack if that helps?

Reply:
Thanks for that observation!